src/unistd.rs

```diff
@@ -192,7 +192,7 @@ impl ForkResult {
 /// ```no_run
 /// use nix::unistd::{fork, ForkResult};
 ///
-/// match fork() {
+/// match unsafe {fork()} {
```

Comment:
You should add a section to the docs describing how to use `fork` safely, like this:

```
/// # Safety
/// 
/// To use this function safely, you must do A, B, and C.
```